### PR TITLE
BYOLLM Design

### PR DESCRIPTION
**Pull request by hikmet-demir**

### CODE REVIEW

File: developer-guides/BYOLLM.mdx [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: "Custom LLM Integration" 
+description: "Guide for using your own LLM or server with ElevenLabs SDK."
```
Comment (louisjoecodes) [inline developer-guides/BYOLLM.mdx:3]: Missing icon

File: developer-guides/BYOLLM.mdx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+
+## Custom LLM Server
+
+To bring a custom LLM server, set up a compatible server endpoint using OpenAI's style, specifically targeting create_chat_completion.
+
+Here's an example server implementation using FastAPI and OpenAI's Python SDK:
+```python
+python
+Copy code
```
Comment (louisjoecodes) [inline developer-guides/BYOLLM.mdx:26]: Is this Copy code supposed to be here

**Review by louisjoecodes — CHANGES_REQUESTED** [review]

File: developer-guides/BYOLLM.mdx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+Run this code or your own server code.
+<img height="200" src="/developer-guides/images/byollm_4.png" />
+
+### Setting Up a Public URL for Your Server
+To make your server accessible, create a public URL using a tunneling tool like ngrok:
+
+```shell
+ngrok http --url=elevenlabs.ngrok.app 8013
```
Comment (louisjoecodes) [inline developer-guides/BYOLLM.mdx:91]: Would remove elevenlabs here and put 'replace with your own url'

File: developer-guides/BYOLLM.mdx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+<img height="200" src="/developer-guides/images/byollm_5.png" />
+
+### Configuring Elevenlabs CustomLLM
+Now let's make the changes on the Elevenlabs
```
Comment (louisjoecodes) [inline developer-guides/BYOLLM.mdx:97]: 'on the elevenlabs' -> 'in ElevenLabs' (or something else)

**Review by louisjoecodes — CHANGES_REQUESTED** [review]

**Comment by hikmet-demir** [comment]
making the changes and updating

**Review by louisjoecodes — APPROVED** [review]

**Pull request merged**